hypervisor/boot/sbl/multiboot.c

```diff
@@ -211,7 +211,7 @@ int init_vm_boot_info(struct acrn_vm *vm)
 		char buf[MAX_BOOT_PARAMS_LEN];
 
 		cmd_dst = kernel_cmdline;
-		cmd_src = hpa2hva((uint64_t)mbi->mi_cmdline);
+		cmd_src = (char *)hpa2hva((uint64_t)mbi->mi_cmdline);
```

Comment:
Do we need a SPACE between (char*) and hpa2hva ?
This is a general coding style question we need to clarify